extensions/notebook/src/dialog/configurePython/configurePythonWizard.ts

```diff
@@ -0,0 +1,217 @@
+				await newPage.onPageEnter();
+			}
+		});
+
+		this._wizard.registerNavigationValidator(async (info) => {
+			let lastPage = pages.get(info.lastPage);
+			let newPage = pages.get(info.newPage);
+
+			// Hit "next" on last page, so handle submit
+			let nextOnLastPage = !newPage && lastPage instanceof PickPackagesPage;
+			if (nextOnLastPage) {
+				let createSuccess = await this.handlePackageInstall();
+				if (createSuccess) {
+					this.showTaskComplete();
+				}
+				return createSuccess;
+			}
+
+			if (lastPage) {
+				let pageValid = await lastPage.onPageLeave();
+				if (!pageValid) {
+					return false;
+				}
+			}
+
+			this.clearStatusMessage();
+			return true;
+		});
+
+		this._wizard.generateScriptButton.hidden = true;
+		this._wizard.pages = [page0, page1];
+		this._wizard.open();
+	}
+
+	public async close(): Promise<void> {
+		await this._wizard.close();
+	}
+
+	public showErrorMessage(errorMsg: string) {
+		this._wizard.message = <azdata.window.DialogMessage>{
+			text: errorMsg,
+			level: azdata.window.MessageLevel.Error
+		};
+	}
+
+	public clearStatusMessage() {
+		this._wizard.message = undefined;
+	}
+
+	private async handlePackageInstall(): Promise<boolean> {
+		let pythonLocation = this.model.pythonLocation;
+		let useExistingPython = this.model.useExistingPython;
+		try {
+			let isValid = await this.isFileValid(pythonLocation);
+			if (!isValid) {
+				return false;
+			}
+
+			if (useExistingPython) {
+				let exePath = JupyterServerInstallation.getPythonExePath(pythonLocation, true);
+				let pythonExists = await utils.exists(exePath);
+				if (!pythonExists) {
+					this.showErrorMessage(this.PythonNotFoundMsg);
+					return false;
+				}
+			}
+		} catch (err) {
+			this.showErrorMessage(utils.getErrorMessage(err));
+			return false;
+		}
+
+		// Don't wait on installation, since there's currently no Cancel functionality
+		let installSettings: PythonInstallSettings = {
+			installPath: pythonLocation,
+			existingPython: useExistingPython,
+			specificPackages: this.model.packagesToInstall
+		};
+		this.jupyterInstallation.startInstallProcess(false, installSettings)
+			.then(() => {
+				this._setupComplete.resolve();
+			})
+			.catch(err => {
+				this._setupComplete.reject(utils.getErrorMessage(err));
+			});
+
+		return true;
+	}
+
+	private async isFileValid(pythonLocation: string): Promise<boolean> {
+		let self = this;
+		try {
+			const stats = await fs.stat(pythonLocation);
+			if (stats.isFile()) {
+				self.showErrorMessage(self.InvalidLocationMsg);
+				return false;
+			}
+		} catch (err) {
+			// Ignore error if folder doesn't exist, since it will be
+			// created during installation
+			if (err.code !== 'ENOENT') {
+				self.showErrorMessage(err.message);
+				return false;
+			}
+		}
+		return true;
+	}
+
+	private showTaskComplete() {
+		this._wizard.registerOperation({
+			connection: undefined,
+			displayName: localize('tableFromFile.taskLabel', 'Create External Table'),
```

Comment:
Wrong display name?

Reply:
Woops. Copied a lot of code over from one of the external table wizards.